alphagov/paas-cf
# [#118843481] Allow different profiles of Cell Instances per environment

## What

[#118843481 - Allow different profiles of Cell Instances per environment](https://www.pivotaltracker.com/story/show/118843481)

We want to define higher number of instances of cells in the production environment only. Environment-specific manifest files have been generated. The Makefile determines what file name is passed to the script which renders the Concourse config file. A Concourse task then includes this file name in the arguments passed to Spruce. 

## How to review

* Edit `paas-cf/manifests/cf-manifest/deployments/env-specific/cf-dev.yml` to define the number of `cell_z*` instances you want to deploy. 
* Check it deploys correct number of instances.
* Run `rspec` from this directory: `paas-cf/manifests/cf-manifest`.

## Who can review

Anyone but @mtekel @dcarley or me.
